tests/backup/backup_test.go

```diff
@@ -140,6 +140,8 @@ var _ = Describe("{CreateMultipleUsersAndGroups}", func() {
 	numberOfGroups := 10
 	users := make([]string, 0)
 	groups := make([]string, 0)
+	userValidation := make([]string, 0)
+	groupValidation := make([]string, 0)
```

Comment:
We don't have to make it so complicated. All you need is one slice each for users and groups. No need for maps.

Reply:
Resolved